spec/acceptance/puppetserver_upgrade_spec.rb

```diff
@@ -16,7 +16,12 @@
 
     case ENV['BEAKER_PUPPET_COLLECTION']
     when 'puppet7'
-      from_version = '7.6.0'
+      from_version =
+        if fact('os.name') == 'Ubuntu' && fact('os.release.major') == '22.04'
+          '7.13.0'
```

Comment:
this makes the upgrade test on Ubuntu 22 essentially a noop, but at least it passes now :D